packages/back-end/src/models/ProjectModel.ts

```diff
@@ -31,15 +31,21 @@ function toInterface(doc: ProjectDocument): ProjectInterface {
   return omit(ret, ["__v", "_id"]);
 }
 
+interface createProjectProps {
```

Comment:
Naming conventions for types should use PascalCase. Can we name it `CreateProjectRequest` to align with the naming we've been doing elsewhere?

Reply:
Sorry, disregard that last feedback re: `Request`—we are in the model.